api/src/extension/cleanup.service.spec.ts

```
@@ -0,0 +1,110 @@
+        SubscriberRepository,
+        AttachmentService,
+        AttachmentRepository,
+        ChannelService,
+      ],
+    });
+    [cleanupService, settingService, helperService] = await getMocks([
+      CleanupService,
+      SettingService,
+      HelperService,
+      ,
+    ]);
```

Biome (1.9.4)

[error] This array contains an empty slot.

Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)